util/scripts/install/install.sh

```diff
@@ -12,18 +12,25 @@ if [[ $? -ne 0 ]]; then
   exit 1
 fi
 
+mkdir logs
+
 # backend
-util/scripts/install/backend.sh
+util/scripts/install/backend.sh > logs/backend.log
```

Comment:
We need to use a different file for the backend install log. `logs/backend.log` is used as the log file for the backend server process while it runs.

Reply:
I changed the logfile name to backend-install.log